### PR TITLE
Bottom banner

### PR DESCRIPTION
**Pull request by ncomerci**
### Desktop
image

### Mobile
image

### CODE REVIEW

File: src/components/Home/BottomBanner/Action.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
+.Action {
+  display: block;
+  background: rgba(255, 255, 255, 0.2);
```
Comment (andyesp) [inline src/components/Home/BottomBanner/Action.css:3]: Can't we reuse one of the blacks here? black-100 or black-200?

File: src/components/Home/BottomBanner/BottomBanner.tsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    title: 'page.home.bottom_banner.newsletter_title',
+    description: 'page.home.bottom_banner.newsletter_description',
+    url: NEWSLETTER_URL,
+  },
+}
+
+function BottomBanner() {
+  const t = useFormatMessage()
+  const [data] = useDclData()
+  const totalFunding = useAbbreviatedNumber(data ? Number(data.funding.total) : 0)
```
Comment (andyesp) [inline src/components/Home/BottomBanner/BottomBanner.tsx:43]: I think you may need to update this or your other PR as you modified this hook 😛

File: src/components/Home/BottomBanner/Stat.css [hunk 3]
```diff
@@ -0,0 +1,18 @@
+.Stat .Stat__Title {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 29px;
+  letter-spacing: -1px;
+  color: white;
```
Comment (andyesp) [inline src/components/Home/BottomBanner/Stat.css:7]: You can use white-900 from the theme here

File: src/components/Home/BottomBanner/Stat.css [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+}
+
+.Stat .Stat__Description {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 14px;
+  letter-spacing: 0.5px;
+  text-transform: uppercase;
+  color: white;
```
Comment (andyesp) [inline src/components/Home/BottomBanner/Stat.css:17]: You can use white-900 from the theme here

File: src/components/Home/BottomBanner/BottomBanner.tsx (hunk 2 above)
Comment (ncomerci) [reply src/components/Home/BottomBanner/BottomBanner.tsx:43]: Yep... haha

**Review by 1emu — APPROVED** [review]

**Review by andyesp — APPROVED** [review]

File: src/components/Home/BottomBanner/BottomBanner.css [hunk 5]
```diff
@@ -0,0 +1,60 @@
+.BottomBanner {
+  background: linear-gradient(180deg, #a524b3 0%, var(--primary) 100%);
+  border-radius: 8px;
+  margin: 10px 0;
+  padding: 24px 15px;
+  width: 100%;
+}
+
+@media screen and (min-width: 768px) {
```
[the remark this answers is not in the document]
Comment (ncomerci) [reply src/components/Home/BottomBanner/BottomBanner.css:9]: makes sense to me 👌🏻

@andyesp thoughts?

[the remark this answers is not in the document]
Comment (andyesp) [reply src/components/Home/BottomBanner/BottomBanner.css:9]: Agree!

**Pull request merged**